rllib/evaluation/rollout_worker.py

```diff
@@ -547,6 +548,36 @@ def make_env(vector_index):
             elif tf1 and policy_config.get("framework") == "tfe":
                 tf1.set_random_seed(seed)
 
+        # Check available number of GPUs.
+        num_gpus = policy_config.get("num_gpus", 0) if \
+            self.worker_index == 0 else \
+            policy_config.get("num_gpus_per_worker", 0)
+        # Error if we don't find enough GPUs.
+        if ray.is_initialized() and \
+                ray.worker._mode() != ray.worker.LOCAL_MODE and \
+                not policy_config.get("_fake_gpus"):
```

Comment:
nice. let's file a feature request for a better way of detecting local mode (on #api-changes)

Reply:
Will do, you don't like `ray.worker._mode() != ray.worker.LOCAL_MODE`? :D